build.gradle.kts

```
@@ -328,12 +328,6 @@ tasks.register("javaioPreCommit") {
   dependsOn(":sdks:java:io:clickhouse:build")
   dependsOn(":sdks:java:io:debezium:expansion-service:build")
   dependsOn(":sdks:java:io:debezium:build")
```

```suggestion
  dependsOn(":sdks:java:io:debezium:build")
  dependsOn(":sdks:java:io:elasticsearch:build")
```

sorry, I meant to remove the compatibility test and remove the basic build target (as other IOs that won't get triggered for java core change)

besides that LGTM

---

I don't understand the comment, because if there is a `build` target it will be triggered. What do you mean other IOs that won't get triggered for java core change?

---

https://github.com/apache/beam/blob/master/.github/workflows/beam_PreCommit_Java_IOs_Direct.yml all the IOs in this list will be triggered for a core change

---

I think I see what you mean. Their larger suite will not be triggered for a core change, but you want one job to run on a core change. I pushed up that change. If flakes continue to be a problem for these core jobs, it might make sense to just not run them in precommit.